components/GitHubEventAction.tsx

```diff
@@ -63,6 +63,16 @@ function GitHubEventAction({type, payload}: GitHubEventActionProps): JSX.Element
       }
       return <span>{action} <ELink link={prURL}>pull request #{prNum}</ELink> in</span>;
     }
+    case 'PullRequestReviewCommentEvent': {
+      const action = payload.action;
+      const prNum = payload.comment.number;
+      const prLink = payload.comment.pull_request_url;
```

Comment:
Note that `pull_request_url` here is the URL to the GitHub API. You probably want the `html_url` to the comment / pull request itself.